jerry-core/parser/js/js-parser-limits.h

```diff
@@ -77,7 +77,7 @@
 #endif /* !PARSER_MAXIMUM_CODE_SIZE */
 
 /**
- * Maximum number of values pushed onto the stack by a function.
+ * Maximum number of values pushed onto the stack by a function.
```

Comment:
There is a 'C2 A0' character here, which a "NO-BREAK SPACE'. Change it to a normal space.